tests/test_data/config/defaults.yml

```diff
@@ -11,9 +11,9 @@ gitlab_base_url: "https://git.example.com/api/v4"
 verify: true
 
 # Auto-generate the issue id in Gitlab. If set to `true`, create a Gitlab issue with
-# the same id as the Bugzilla ticket number.  The group members which are 
```

Comment:
These whitespace changes are inconsequential -- just a feature of my code editor